### DISCUSSION TITLE
[Feature] [App] As a share to destination

### DISCUSSION THREAD

**Discussion by maicss** [immich-app/immich#10743] [Feature Request]
### I have searched the existing feature requests to make sure this is not a duplicate request.

- [X] Yes

### The feature

I have an album called `Wallpaper`, to collect some beautiful pictures I meet on the internet.

I find a picture on twitter/telegram, long press the picture and click share, a lot of app is listed, but not immich.

Feature: as a share to client, and choose which album to save. And if support video is even better.

Thanks for your great work!

### Platform

- [ ] Server
- [ ] Web
- [X] Mobile

**Comment by bo0tzz** [comment]
#2659
